docs/images/esm.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="82" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><mask id="a"><rect width="82" height="20" rx="3" fill="#fff"/></mask><g mask="url(#a)"><path fill="#555" d="M0 0h33v20H0z"/><path fill="#e05d44" d="M33 0h49v20H33z"/><path fill="url(#b)" d="M0 0h82v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="16.5" y="15" fill="#010101" fill-opacity=".3">esm</text><text x="16.5" y="14">esm</text><text x="56.5" y="15" fill="#010101" fill-opacity=".3">3.61 kB</text><text x="56.5" y="14">3.61 kB</text></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="82" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><mask id="a"><rect width="82" height="20" rx="3" fill="#fff"/></mask><g mask="url(#a)"><path fill="#555" d="M0 0h33v20H0z"/><path fill="#e05d44" d="M33 0h49v20H33z"/><path fill="url(#b)" d="M0 0h82v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="16.5" y="15" fill="#010101" fill-opacity=".3">esm</text><text x="16.5" y="14">esm</text><text x="56.5" y="15" fill="#010101" fill-opacity=".3">3.65 kB</text><text x="56.5" y="14">3.65 kB</text></g></svg>
```

Comment:
Are you following eslint? Cause you should do the new line thing if you are

Reply:
this is auto generated.
Having it split into multiple lines would be nice and make future diffs significantly smaller (we should just be updating the size difference).

Either way, I'll make an issue to look into it later.

Reply:
https://github.com/Tram-One/tram-one/issues/70